public/app/features/alerting/unified/components/rules/RuleActionsButtons.tsx

```diff
@@ -145,15 +145,17 @@ export const RuleActionsButtons = ({ rule, rulesSource }: Props) => {
 
     if (isGrafanaRulerRule(rulerRule) && canReadProvisioning) {
       moreActions.push(<Menu.Item label="Export" icon="download-alt" onClick={toggleShowExportDrawer} />);
-
       if (config.featureToggles.alertingModifiedExport) {
         moreActions.push(
           <Menu.Item
             label="Modify export"
             icon="edit"
             onClick={() =>
               locationService.push(
-                `/alerting/${encodeURIComponent(ruleId.stringifyIdentifier(identifier))}/modify-export`
+                urlUtil.renderUrl(
```

Comment:
I'd suggest using `createUrl` function because it takes care of the subpath prefix (when grafana is hosted on a subpath)

Reply:
Good point!